cds-snc/cppd-medical-report
# File Uploader improvements

# Description

File Uploader improvements:
- improved styling
- enable remove link

# Test Instructions

- Navigate to add_condition
- Click Upload button and select a file
- Notice the lovely styling
- Try out the remove link
